pipelines/update_exposure_tercile.py

```diff
@@ -0,0 +1,120 @@
+import sys
+from datetime import datetime, timedelta
+
+import numpy as np
+import pandas as pd
+from sqlalchemy import text
+
+from src.utils import database
+
+ROLL_WINDOW = 7
```

Comment:
We also specify this [here](https://github.com/OCHA-DAP/ds-floodexposure-monitoring-app/blob/ef56d92cc1856510e2db0d3e09304bb897ef516d/constants.py#L11). I'm not sure how best to standardize across both repos but we should be aware it's duplicated if we ever adjust.

Reply:
Well noted yeah. Could be a use for a GH organizational variable because it's not sensitive, and we want to be able to see it. Still not sure if it's the best way though.